### PR TITLE
add metadata link to mobile optimised pages

### PR DESCRIPTION
**Pull request by johnduffell**
@NataliaLKB just a bit of tidyup and adding in the link

### CODE REVIEW

File: dev-build/conf/routes [hunk 1]
```diff
@@ -355,5 +355,5 @@ GET         /embed/video/*path
 
 # Articles
 GET         /*path.json                                                                                              controllers.ArticleController.renderArticle(path, lastUpdate: Option[String], rendered: Option[Boolean])
-GET         /*path/amp                  controllers.ArticleAMPController.renderPath(path)
+GET         /*path/amp                                                                                               controllers.ArticleController.renderArticle(path, lastUpdate: Option[String], rendered: Option[Boolean])
 GET         /*path                                                                                                   controllers.ArticleController.renderArticle(path, lastUpdate: Option[String], rendered: Option[Boolean])
```
Comment (janua) [inline dev-build/conf/routes]: How does this... work?

**Comment by janua** [comment]
:+1:
Is there any reason you used `request.isAmp` instead of leaving it to the `routes` file which has already matched out the fact that it ends with `/amp`?

**Comment by NataliaLKB** [comment]
@johnduffell just pulled this down and went to looked at [this](http://www.theguardian.com/commentisfree/2015/sep/10/africa-rising-villages-left-behind-nigeria/amp) article. Appears broken to me on `dev-build`. Lets review together in the morning, as I likely missed something.

**Comment by NataliaLKB** [comment]
Ignore my last comment. Just needed to run `clean`. Looks great :+1:

**Comment by johnduffell** [comment]
@janua I will refactor it all to use the routes file properly one day... :stuck_out_tongue_winking_eye:

**Pull request merged**